scripts/revive.toml

```diff
@@ -25,11 +29,5 @@
     Disabled = true
 [rule.import-shadowing]
     Disabled = true
-[rule.exported]
-    Disabled = true # TODO: Enable after existing issues are fixed
-[rule.redefines-builtin-id]
-    Disabled = true # TODO: Enable after existing issues are fixed
-[rule.package-comments]
-    Disabled = true # TODO: Enable after existing issues are fixed
-[rule.indent-error-flow]
-    Disabled = true # TODO: Enable after existing issues are fixed
```

Comment:
Could we keep this PR focused on that change and address the other removals in a separate PR? This will make it easier to track the history of these changes in the future.

Reply:
I think its fine to just enable the remaining ones with this PR since at this time we are sure there are no more existing issues to be fixed

Reply:
Modifying the PR title and merge commit to specify the PR also enable remaining disabled rules